cmd/opentelemetry/app/internal/esclient/index_name.go

```diff
@@ -12,39 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package esspanreader
+package esclient
```

Comment:
It should go to a different package than `esclient`. The `esclient` is dedicated only for the ES client used by Jaeger writer/reader.

Reply:
moved to  `./internal/esutil`